network/peer-monitoring-service/client/src/logging.rs

```diff
@@ -46,6 +46,9 @@ pub enum LogEntry {
     NodeInfoRequest,
     PeerMonitorLoop,
     SendRequest,
+
+    #[cfg(feature = "network-perf-test")] // Disabled by default
+    PerformanceMonitoringRequest,
```

Comment:
Ughh... I can't think of a use-case. Nodes without support for the feature will simply drop the performance messages because the message enum won't have an entry in them. So, this will essentially be a serialization failure.